### ISSUE TITLE
New Maryland Three-inch areas to add to the GeoJSON MultiPolygon

### ISSUE THREAD

**Issue by talllguy** [osmlab/editor-layer-index#445]
There are two new coverage areas in the Maryland Three-inch imagery dataset, collected in 2017. The state updates the imagery at the endpoint so the service URL is still the same as it is in #171. These just launched in the last week.

> The imagery for this service is composed of imagery flown in 2014 (Rockville), 2016 (Easton), and 2017 (Annapolis, Gaithersburg, Port of Baltimore, Westminster).

_Quoted from the metadata_

In pull #206 the special-case multipolygon GeoJSON was discussed. It appears the current editor index is missing these new areas' shapes:
![image](https://user-images.githubusercontent.com/1827881/37381150-06296b06-2712-11e8-878b-188b2ce5b537.png)

**Comment by grischard** [comment]
Hi @talllguy, thank you for this! Would you please have polygons for these areas?

**Comment by cicku** [comment]
@talllguy Hi, this one has been included for a while. However, recent test shows that it is blank. Is the source removed? Also, since iMAP was refreshed in the last 2 years, is 3 inch still a good source here? I'm inclined to remove it actually.

**Comment by talllguy** [comment]
@cicku you are right, it appears down. It should still be working; I will check with IMAP.

**Comment by PvtPuddles** [comment]
What is the status on this?  In iD, I can move the camera to a location that previously had 3-inch imagery and then manually move the camera back to these new locations to see the imagery.

It would be nice if the 3-inch imagery showed up in the dropdown list.

Since this issue was opened, MD 3-inch imagery seems to be available in much more of the state, including an expanded area around Baltimore.

<img width="940" height="526" alt="Image" src="https://github.com/user-attachments/assets/85ba2d72-83b0-4174-8f14-6147fc582ee6" />

<img width="829" height="400" alt="Image" src="https://github.com/user-attachments/assets/4b1b8155-19c8-4d99-90d5-7c2bbc0ab4bd" />
